File: ONLINE_CODING_TEST/school.programmers/lvl_2/12951_JadenCase.java
class Solution {
  public String solution(String s) {
    StringBuilder sb = new StringBuilder();
    boolean flag = true;
    s = s.toLowerCase();
    
    for (char c : s.toCharArray()) {
      if (flag) {
        c = Character.toUpperCase(c);
      }
      sb.append(c);
      
      flag = Character.isWhitespace(c) ? true : false;
    }
    
    return sb.toString();
  }
}

File: ONLINE_CODING_TEST/school.programmers/lvl_2/12951_JadenCase.js
function solution(s) {
  s = s.toLowerCase();
  var answer = '';
  let flag = true;

  for (let c of s.split('')) {
    if (flag) c = c.toUpperCase();

    answer += c;

    flag = (c === ' ') ? true : false;
  }

  return answer;
}

const tests = ["3people unFollowed me", "for the last week"];

for (const test of tests) {
  console.log('-'.repeat(100));
  console.log('param : ', test, '\n');
  console.time();
  console.log('\n\t->\t', solution(test));
  console.timeEnd();
  console.timeLog();
  console.log('-'.repeat(100) + '\n');
}